src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -596,6 +654,36 @@ export default class StripViewModel extends BaseModel {
         return SELECTORS.CLASSNAMES.ARRIVAL;
     }
 
+    /**
+     * Returns the runway and if it is expected or assigned based on whether an
+     * aircraft is a `departure` or an `arrival`.
+     *
+     * @for AircraftStripView
+     * @method _buildRunwayInformation
+     * @param aircraftModel {AircraftModel}
+     * @return {object}
+     */
+    _buildRunwayInformation(aircraftModel) {
+        if (aircraftModel.isDeparture()) {
+            return {
+                assigned: aircraftModel.pilot.hasDepartureClearance,
+                name: aircraftModel.fms.departureRunwayModel.name
+            };
+        }
+
+        if (aircraftModel.isArrival() !== null) {
```

Comment:
Just `if (aircraftModel.isArrival()) {`. This will always return a boolean.

Reply:
true...copy-n-paste-fail